packages/engine/paima-sm/src/cde-erc721-transfer.ts

```diff
@@ -21,7 +31,26 @@ export default async function processErc721Datum(
     );
     const newOwnerData = { cde_id: cdeId, token_id: tokenId, nft_owner: toAddr };
     if (ownerRow.length > 0) {
-      updateList.push([cdeErc721UpdateOwner, newOwnerData]);
+      if (isBurn) {
+        if (cdeDatum.burnScheduledPrefix) {
+          const scheduledInputData = `${cdeDatum.burnScheduledPrefix}|${ownerRow[0].nft_owner}|${tokenId}`;
+          const scheduledBlockHeight = isPresync
```

Comment:
This behavior for handling presync looks different than what we have in `cde-erc20-deposit`

Reply:
Hmmm, right... It's more consistent to not schedule data during the presync at all in this PR.